ONSdigital/eq-deploy
# Adding Jenkins DSL for our jobs

**What**
Converting our Jenkins jobs to the DSL

**How to test**
1. Create a Jenkins seed job (see guide: https://github.com/jenkinsci/job-dsl-plugin/wiki/Tutorial---Using-the-Jenkins-Job-DSL)
2. Run the DSL jobs in the jobs directory

**Who can review**
Anyone apart from @warrenbailey
